pysd/py_backend/statefuls.py

```diff
@@ -1583,7 +1650,7 @@ def run(self, params=None, return_columns=None, return_timestamps=None,
         --------
         >>> model.run(params={'exogenous_constant': 42})
         >>> model.run(params={'exogenous_variable': timeseries_input})
-        >>> model.run(return_timestamps=[1, 2, 3.1415, 4, 10])
+        >>> model.run(return_timestamps=[1, 2, 3, 4, 10])
```

Comment:
are we still handling returns of arbitrary times, or just multiples of the timestep?

Reply:
From 2.x we only support multiples of the time step. I have added in this branch a warning message when an arbitrary value is not included in the output, so the user will be aware of the problem.